### PR TITLE
Require nokogiri 1.8.5 or greater

### PR DESCRIPTION
**Pull request by discordier**
Addresses CVE-2018-14404
https://nvd.nist.gov/vuln/detail/CVE-2018-14404

### CODE REVIEW

**Comment by discordier** [comment]
Unsure how to fix the "line too long" issue in build.
Shall the comment be moved to the line before or shall the line be wrapped?

**Comment by shiftkey** [comment]
@discordier 👍 to moving the comment to the preceding line

**Comment by discordier** [comment]
Fixed and force pushed

File: github-pages.gemspec [hunk 1]
```diff
@@ -26,7 +26,8 @@ Gem::Specification.new do |s|
 
   s.add_dependency("mercenary", "~> 0.3")
   s.add_dependency("terminal-table", "~> 1.4")
-  s.add_dependency("nokogiri", ">= 1.8.2", "< 2.0") # CVE-2017-9050, CVE-2017-18258
+  # CVE-2017-9050, CVE-2017-18258, CVE-2018-14404
```
Comment (parkr) [inline github-pages.gemspec]: Let’s actually just remove the comment, if you don’t mind. The Git history for this line will have the context :)

Comment (discordier) [reply github-pages.gemspec]: lol, ok then :smile_cat:

**Pull request merged**